### ISSUE TITLE
Error calling 'infix': symbol `weather` not found

### ISSUE THREAD

**Issue by jkleiser** [glycerine/zygomys#31]
I just installed zygo v5.0.1 on my Mac (macOS 10.14.1), and when I try the example I found in the wiki
```
(def sn (snoopy asst: (hornet forecast: (weather desc:"sunny"))))
```
then I get
```
error in evalGeneratedFunction:4: Error calling 'infix': symbol `weather` not found
in infix:0
in __main:22
```
What may be the problem here?

**Comment by glycerine** [comment]
Yeah. Sorry this isn't clearer. It's perfectly natural to wonder why this would work at all. After all. Without a `defmap` or registration of a Go struct, none of the `snoopy`, `hornet`, or `weather` calls should work.  Nobody has defined those functions.

So what is going on. Well, there is a special flag for demo and testing purposes.

https://github.com/glycerine/zygomys/blob/master/zygo/cfg.go#L43

So the trick is, starting zygo with `zygo -demo` will register those functions, letting the example function.
